README.md

```diff
@@ -67,7 +58,11 @@ Forutsetter at man har tilgang til git repository tilgjengelig på internt nett
 
  ```bash
  heroku create en-kul-ny-feature
- ./heroku-build.sh
+ git add .
+ git commit -m "[WIP]"
+ git push
```

Comment:
Trenger ikke å pushe til GitHub for å kjøre `./heroku-build.sh`; det holder at commit-en finnes lokalt. (Hvis jeg forstod riktig.)